src/chapter_2/installation.md

```diff
@@ -35,4 +36,26 @@ Now verify the CLI is installed properly.
 
 ```
 anchor --version
-```
\ No newline at end of file
+```
+
+### Installing using Anchor version manager (avm)
+
+Anchor version manager is a tool for using multiple versions of the anchor-cli. It will require the same dependencies as building from source. It is recommended you uninstall the NPM package if you have it installed.
+
+Install `avm` using Cargo. Note this will replace your `anchor` binary if you had one installed.
+
+```
+cargo install --git https://github.com/project-serum/anchor avm --locked --force
+```
+
+Install the latest version of the CLI using `avm`. You can also use the command to upgrade to the latest release in the future.
+
+```
+avm use latest
+```
+
+Verify the installation.
+
+```
+anchor --version
+```
```

Comment:
this sounds like it returns the anchor version that avm is currently using but it actually returns the avm version

Reply:
it returns the anchor version?

```
~/Code
❯ avm --version
avm 0.21.0

~/Code
❯ anchor --version
anchor-cli 0.21.0

~/Code
❯ avm use 0.20.0

~/Code
❯ anchor --version
anchor-cli 0.20.0

~/Code
❯ avm use 0.19.0

~/Code
❯ anchor --version
anchor-cli 0.19.0
```

Reply:
it didnt for me but forgot to save terminal session. I just tried to reproduce and found sth else:
Im currently on version `0.21.0` which is also my only installed version. When I do `avm use 0.20.1`, which I do not have installed, it asks me whether I want to install it. If I say no, it still changes the current version to `0.20.1` instead of saying on `0.21.0`. Is this intentional?

Reply:
no that is not intentional

Reply:
You'd expect `anchor --version` to display the current avm version? That seems odd to me.

`nvm --version` shows the currently installed version of nvm, which is the same functionality as `avm --version`. This is distinct from the version of node.

```
❯ nvm --version
nvm version 1.0.2
```

```
❯ node --version
v16.13.0
```

Reply:
sorry. nvm this. I just misread and mistyped the commands 🤦‍♂️